src/util/fleetCost.js

```diff
@@ -0,0 +1,21 @@
+import shipCosts from "./enum/shipCosts.js";
+
+/**
+ *
+ * @param {Array<number>} ships - Array of ship quantity per IDs to sum costs for.
+ * @return {Array<number>} sum of cost per resource [M / C / D]
+ */
+function fleetCost(ships) {
+  const fleetRes = [0, 0, 0];
```

Comment:
Can you use explicit keys instead of array

and when you return, just use Object.values(fleetRes) ?